packages/editor-ui/src/components/ValueSurvey.vue

```diff
@@ -0,0 +1,274 @@
+
+const DEFAULT_TITLE = `How likely are you to recommend n8n to a friend or colleague?`;
+const GREAT_FEEDBACK_TITLE = `Great to hear! Can we reach out to see how we can make n8n even better for you?`;
+const DEFAULT_FEEDBACK_TITLE = `Thanks for your feedback! We'd love to understand how we can improve. Can we reach out?`;
+
+export default mixins(workflowHelpers).extend({
+	name: 'ValueSurvey',
+	components: {
+		ModalDrawer,
+	},
+	computed: {
+		getTitle(): string {
+			if (this.form.value !== '') {
+				if (Number(this.form.value) > 7) {
+					return GREAT_FEEDBACK_TITLE;
+				} else {
+					return DEFAULT_FEEDBACK_TITLE;
+				}
+			} else {
+				return DEFAULT_TITLE;
+			}
+		},
+		isEmailValid(): boolean {
+			return VALID_EMAIL_REGEX.test(String(this.form.email).toLowerCase());
+		},
+	},
+	data() {
+		return {
+			form: {
+				email: '',
+				value: '',
+			},
+			showButtons: true,
+			VALUE_SURVEY_MODAL_KEY,
+		};
+	},
+	methods: {
+		closeDialog(): void {
+			if (this.form.value === '') {
+				this.$telemetry.track('User responded value survey score', {
+					instance_id: this.$store.getters.instanceId,
+					nps: '',
+				});
+			} else {
+				this.$telemetry.track('User responded value survey email', {
+					instance_id: this.$store.getters.instanceId,
+					email: '',
+				});
+			}
+
+			this.$store.commit('ui/closeTopModal');
+		},
+		onInputChange(value: string) {
+			this.form.email = value;
+		},
+		selectSurveyValue(value: string) {
+			this.form.value = value;
+			this.showButtons = false;
+			this.$store
+				.dispatch('settings/submitValueSurvey', { value: this.form.value })
+				.then((response: IN8nPromptResponse) => {
+					if (response.updated) {
+						this.$telemetry.track('User responded value survey score', {
+							instance_id: this.$store.getters.instanceId,
+							nps: this.form.value,
+						});
+					}
+				});
+		},
+		send(): void {
+			if (this.isEmailValid) {
+				this.$store
+					.dispatch('settings/submitValueSurvey', {
+						email: this.form.email,
+						value: this.form.value,
+					})
+					.then((response: IN8nPromptResponse) => {
+						if (response.updated) {
+							this.$telemetry.track('User responded value survey email', {
+								instance_id: this.$store.getters.instanceId,
+								email: this.form.email,
+							});
+							this.$showMessage({
+								title: 'Thanks for your feedback',
+								message: `If you’d like to help even more, answer this <a target="_blank" href="https://n8n-community.typeform.com/quicksurvey#nps=${this.form.value}&instance_id=${this.$store.getters.instanceId}">quick survey.</a>`,
+								type: 'success',
+								duration: 15000,
+							});
+						}
+						setTimeout(() => {
+							this.form.value = '';
+							this.form.email = '';
+							this.showButtons = true;
+						}, 1000);
+						this.$store.commit('ui/closeTopModal');
+					});
+			}
+		},
+	},
+	mounted() {
+		this.$telemetry.track('User shown value survey', {
```

Comment:
this only happens once when the page is loaded because of keep alive.. so this is wrong..

Reply:
Done. Added isActive prop as suggested